File: Frontend/bistroflow/src/app/Models/userordercount.ts
export class UserOrderCount {
    userId: number;
    userName: string;
    ordersSold: number;

    constructor(userId: number = 0, userName: string = '', ordersSold: number = 0) {
        this.userId = userId;
        this.userName = userName;
        this.ordersSold = ordersSold;
      }
}

File: Backend/bistroflow/src/main/java/com/bistroflow/model/UserOrderCount.java
package com.bistroflow.model;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
public class UserOrderCount {
    @Id
    private Long userId;
    private String userName;
    private Long ordersSold;


}
